Microsoft.Toolkit.Uwp.UI.Animations/ParallaxService.cs

```diff
@@ -0,0 +1,112 @@
+        public static FrameworkElement GetScrollingElement(DependencyObject obj)
+        {
+            return (FrameworkElement)obj.GetValue(ScrollingElementProperty);
+        }
+
+        /// <summary>
+        /// Sets the element that is, or contains, a ScrollerViewer.
+        /// </summary>
+        /// <param name="obj">The object to set the value on.</param>
+        /// <param name="value">The element that is, or contains a ScrollViewer.</param>
+        public static void SetScrollingElement(DependencyObject obj, FrameworkElement value)
+        {
+            obj.SetValue(ScrollingElementProperty, value);
+        }
+
+        /// <summary>
+        /// Gets a value for how fast the parallax effect should be.
+        /// </summary>
+        /// <param name="obj">The object to get the value from.</param>
+        /// <returns>A value for how fast the parallax effect should be.</returns>
+        public static double GetMultiplier(DependencyObject obj)
+        {
+            return (double)obj.GetValue(MultiplierProperty);
+        }
+
+        /// <summary>
+        /// Sets the value for how fast the parallax effect should be.
+        /// </summary>
+        /// <param name="obj">The object to set the value on.</param>
+        /// <param name="value">The value for how fast the parallax effect should be.</param>
+        public static void SetMultiplier(DependencyObject obj, double value)
+        {
+            obj.SetValue(MultiplierProperty, value);
+        }
+
+        private static void OnScrollingElementChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            CreateParallax(d as UIElement, GetScrollViewer(d), (double)d.GetValue(MultiplierProperty));
+        }
+
+        private static void OnMultiplierChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            CreateParallax(d as UIElement, GetScrollViewer(d), (double)d.GetValue(MultiplierProperty));
+        }
+
+        private static ScrollViewer GetScrollViewer(DependencyObject obj)
+        {
+            var element = obj.GetValue(ScrollingElementProperty) as DependencyObject;
+            var scroller = element as ScrollViewer;
+            return scroller ?? element?.FindDescendant<ScrollViewer>();
+        }
+
+        private static void CreateParallax(UIElement parallaxElement, ScrollViewer scroller, double multiplier)
+        {
+            if ((parallaxElement == null) || (scroller == null))
+            {
+                return;
+            }
+
+            CompositionPropertySet scrollerViewerManipulation = ElementCompositionPreview.GetScrollViewerManipulationPropertySet(scroller);
+
+            Compositor compositor = scrollerViewerManipulation.Compositor;
+
+            ExpressionAnimation expression = compositor.CreateExpressionAnimation(
+                "Matrix4x4.CreateFromTranslation(Vector3(0.0f, ParallaxMultiplier * scroller.Translation.Y, 0.0f))");
```

Comment:
The parallax effect is fixed on the Y axe. We can't use the service for an horizontal parallax effect with a FlipView for example. May be an enum with the X or Y axe used (may be both) could be added.